pepper/templates/emails/application_decisions/waitlisted.html

```diff
@@ -2,42 +2,41 @@
 {% block message %}
     <tr>
       <td class="content-block">
-        Hi {{ user.fname }} {{ user.lname }},
+        Hey {{ user.fname }},
       </td>
     </tr>
     <tr>
       <td class="content-block">
-        Thank you for taking the time to apply to {{ config.HACKATHON_NAME }}. Your application has been waitlisted.
+        Thank you for taking the time to apply to {{ config.EVENT_NAME }}. Unfortunately, we had many more applicants than we have 
+        space for, and we are unable to offer you a spot for this year. However, we're still hoping that we can fit you in, so we 
+        have made you eligible for our <strong>day-of waitlist</strong>.
       </td>
     </tr>
     <tr>
       <td class="content-block">
-        Applications for {{ config.HACKATHON_NAME }} are first-come, first-serve for teams and randomized for individuals. This decision is not a reflection of your resume or your experience level. We simply take that information to run our own internal metrics and share your resume with companies who are recruiting. Cross your fingers though! There will be a second round of acceptances on October 17th. If you live in the Austin metro area, we encourage you to try to walk-in on the day of {{ config.HACKATHON_NAME }}, {{ config.EVENT_START }}.
+        This decision is not a reflection of your resume or your experience level. While wish we could accept everyone who applied, 
+        there's a limited amount of space and it wouldn't be fun for anyone if it got too crowded. But there's still hope - on 
+        {{ config.EVENT_START }}, we will be announcing whether we still have spots open for {{ config.EVENT_NAME }} <strong>after 
+        accepted attendee check-in closes at noon. </strong> If so, then we will begin taking walk-ins at this time. Keep an eye out 
+        for an announcement on our website (via a banner), via our social media, or email!
```

Comment:
this is good for our final waitlist message but del for our first wave waitlist, would be redundant and feel less personal if we included the same message in both waitlist emails lol.